eva_submission/ENA_submission/json_to_ENA_json.py

```diff
@@ -18,11 +18,16 @@ def today():
 
 class EnaJsonConverter(AppLogger):
 
-    def __init__(self, submission_id, input_eva_json, output_folder, output_file_name):
+    def __init__(self, submission_id, input_eva_json, output_folder, output_file_name, existing_project_accession=None):
```

Comment:
I was a bit surprised to see this change since we have another ticket about supporting existing project accessions, but I guess since the JSON path [doesn't support the old way anyway](https://github.com/EBIvariation/eva-submission/blob/02918cf988e546191a95a7fcb55cb8f4cd40f8fc/eva_submission/eload_submission.py#L124) it's probably fine.